mlos_bench/mlos_bench/tests/config/tunable-values/tunable-values-example.jsonc

```diff
@@ -2,8 +2,8 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/MLOS/main/mlos_bench/mlos_bench/config/schemas/tunables/tunable-values-schema.json",
 
-    "sched_migration_cost_ns": 500000,
-    "sched_latency_ns": 12000000,
+    "sched_migration_cost_ns": 400000,
+    "sched_latency_ns": 10000000,
```

Comment:
Are these changing to ensure they?

Reply:
...to ensure they are different from the default values that we have in `tunables.json`

Reply:
Sorry, seems my comment got lost.  Yes, that's what I was asking for.  Can you just add a comment to that effect so we know why.

Reply:
I added the comment in the suggestion above.